glide.yaml

```diff
@@ -25,3 +21,4 @@ import:
   version: ~1.4.0
   subpackages:
   - pkg/api/resource
+- package: github.com/abh/errorutil
```

Comment:
It wasn't created automatically, so I don't actually know.  Without any branches or tags to specify an actual version, I'm not sure what it would buy us over the lock file.